packages/cosmic-swingset/app/app.go

```diff
@@ -497,6 +498,19 @@ func (app *AgoricApp) SimulationManager() *module.SimulationManager {
 	return app.sm
 }
 
+// MakeCodecs constructs the *std.Codec and *codec.Codec instances used by
```

Comment:
This constructor is maintaining a global registry. Would it be safer to construct codecs in an `init` func and expose the single app codec in global scope? As much as I dislike building atop the global registry pattern, I think it would be safer to have globals than having a function that can only be safely called once.

Reply:
Thanks for your comment.

I don't like ignoring these valuable insights, but I feel like my hands are tied.  I'm merging changes directly from the Gaia repo (https://github.com/cosmos/gaia) so that I'm prepared to merge future changes without much effort, too.

If you want to file an issue there describing your suggestion and motivation from a seasoned Golang programmer's perspective, that could have some impact since they tend to refactor things every few weeks.